src/main/java/org/spongepowered/api/entity/ai/task/builtin/creature/target/FindNearestAttackableTargetAITask.java

```diff
@@ -24,31 +24,39 @@
  */
 package org.spongepowered.api.entity.ai.task.builtin.creature.target;
 
+import org.spongepowered.api.entity.living.Creature;
 import org.spongepowered.api.entity.living.Living;
 
 import java.util.function.Predicate;
 
-public interface FindNearestAttackableTargetAITask extends TargetAITask<FindNearestAttackableTargetAITask> {
+public interface FindNearestAttackableTargetAITask<
```

Comment:
Run the line to length then break...don't do this barely half-line break.

Common sense > auto-formatter.